auth0/token.go

```diff
@@ -74,7 +75,7 @@ func NewAuth0Client(env,
 }
 
 // GetToken ...
-func (a *ClientProvider) GetToken() (string, error) {
+func (a *ClientProvider) GetToken(input bool) (string, error) {
```

Comment:
the input is useless after caching the cert, no need for it

Reply:
done!